devito/interfaces.py

```diff
@@ -402,7 +414,10 @@ def __init__(self, *args, **kwargs):
             self.initializer = kwargs.get('initializer', None)
             if self.initializer is not None:
                 assert(callable(self.initializer))
-            self.numa = kwargs.get('numa', False)
+            first_touch = kwargs.get('first_touch', None)
+            if first_touch is None:
+                first_touch = Global_First_Touch
+            self.first_touch = first_touch
```

Comment:
these 4 lines could probably be rewritten as:
``self.first_touch = kwargs.get('first_touch', Global_First_Touch)``